mem0/memory/telemetry.py

```diff
@@ -58,7 +60,8 @@ def capture_event(event_name, memory_instance, additional_data=None):
     if additional_data:
         event_data.update(additional_data)
 
-    telemetry.capture_event(event_name, event_data)
+    if TELEMETRY:
+        telemetry.capture_event(event_name, event_data)
```

Comment:
Instead of doing this, you can directly do something like this:

```python
if not MEM0_TELEMETRY:
    self.posthog.disabled = True
```

This will automatically make sure that none of the events get sent to the server.

Reply:
Drink and code accident, added it again